src/store/tree.module.js

```diff
@@ -0,0 +1,118 @@
+/**
+ * Copyright (C) NIWA & British Crown (Met Office) & Contributors.
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ */
+import { clear } from '@/components/cylc/tree'
+import applyDeltasLookup from '@/components/cylc/common/deltas'
+import Alert from '@/model/Alert.model'
+import applyDeltasTree from '@/components/cylc/tree/deltas'
+
+const state = {
```

Comment:
I've split the `workflows.module` since it had data used by GScan, and also by the Tree view. The tree view data is relatively more complex at this point, so moving it out simplifies the `workflows.module`.